scripts/new-adr.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+slug=$1
+
+if [ -z  "${npm_lifecycle_event}" ]
+then
+  echo "This script must be invoked by \`npm run new-adr\` and not run directly."
+  exit 1
+fi
+
+if [ -z "${slug}" ]
+then
+  echo "--slug'my-example-adr-name' needs to be passed in when running \`npm run new-adr\`"
+  exit 1
+fi
+
+num_prefix=$(printf '%04d' "$(find docs/adrs -iname '*.md' | wc -l)")
+
+new_adr_path="./docs/adrs/${num_prefix}-${slug}.md"
+
+cat << ADRHD > "${new_adr_path}"
+---
+title: '${num_prefix} A title for this ADR'
+description: |
+  A description for ADR ${num_prefix}
```

Comment:
**suggestion (non-blocking)**: This doesn't seem to be part of the regular template, but I wonder if we should add it, for anyone that doesn't use this script. I would say that could be a separate PR except that would affect the `tail` command below, so it might be good to do it all in one PR.

Reply:
Thanks for the suggestion @felipe-lee I'll add it to this PR.